src/clusterfuzz/_internal/google_cloud_utils/batch.py

```diff
@@ -281,10 +276,8 @@ def _get_spec_from_config(command, job_name):
     config_name += '-PREEMPTIBLE'
   else:
     config_name += '-NONPREEMPTIBLE'
-  # TODO(metzman): Get rid of this when we stop doing privileged operations in
-  # utasks.
-  if command in _UNPRIVILEGED_TASKS:
-    config_name += '-UNPRIVILEGED'
+  # TODO(metzman): Remove this since all jobs will be unprivileged.
```

Comment:
I'm not removing UNPRIVILEGED right away because it could be unsafe if I were to do this without landing the rename in https://team-review.git.corp.google.com/c/gosst/clusterfuzz-config/+/2350609